pkg/admission/decode.go

```diff
@@ -54,3 +55,31 @@ func DecodeCloudProfileConfig(decoder runtime.Decoder, config *runtime.RawExtens
 
 	return cloudProfileConfig, nil
 }
+
+// DecodeBackupBucketConfig decodes the `BackupBucketConfig` from the given `RawExtension`.
+func DecodeBackupBucketConfig(decoder runtime.Decoder, config *runtime.RawExtension) (*gcp.BackupBucketConfig, error) {
```

Comment:
Unit test missing. Same for the function below

Reply:
Unit tests are added